src/ui/server/helpers/summary-lists/company-house-summary-list.ts

```diff
@@ -43,7 +43,7 @@ const generateAddressHTML = (address: object): string => {
  * @returns {String} Sic codes as a single string or default empty string
  */
 const generateSicCodesValue = (sicCodes: Array<string>): string => {
-  if (sicCodes.length) {
+  if (sicCodes && sicCodes.length) {
```

Comment:
Can we update the unit test for `generateSicCodesValue` to check if there is no `sicCodes` provided? It currently only checks for empty array 

`sicCodes` param should now be optional